vcpkg-init/mint-standalone-bundle.ps1

```diff
@@ -1,13 +1,22 @@
-[CmdletBinding()]
+[CmdletBinding(PositionalBinding=$False)]
 Param(
-    [Parameter(Mandatory = $true)]
+    [Parameter()]
     [string]$DestinationTarballName,
-    [Parameter(ValueFromRemainingArguments = $true)]
-    [string[]]$AdditionalFiles,
     [Parameter()]
-    [string]$TempPath
+    [string]$DestinationDir,
+    [Parameter(Mandatory)]
+    [string]$TempPath,
+    [Parameter()]
+    [switch]$readonly,

```

Comment:
Why is this configurable? All callers pass `$True` afaict.

Reply:
The "one liner" version passes false.